### DISCUSSION TITLE
Payload _binary_headers encoding

### DISCUSSION THREAD

**Discussion by MaxKingPor** [aio-libs/aiohttp#8034] [Q&A]
Whether the encode in `Payload._binary_headers` should be replaced with `self._encoding`
![image](https://github.com/aio-libs/aiohttp/assets/43533178/770e3ef5-13bf-4f48-8afd-bfb92850d7bd)

**Comment by Dreamsorcerer** [comment]
No, see https://github.com/aio-libs/aiohttp/issues/7943#issuecomment-1838705607
